src/libs/OptionsListUtils.ts

```diff
@@ -1867,6 +1974,10 @@ function filterAndOrderOptions(options: Options, searchInputValue: string, confi
         filteredReports = filteredReports.slice(0, config.maxRecentReportsToShow);
     }
 
+    if (typeof config?.maxRecentReportsToShow === 'number' && filteredWorkspaceChats && filteredWorkspaceChats?.length > config.maxRecentReportsToShow) {
+        filteredWorkspaceChats = filteredWorkspaceChats.slice(0, config.maxRecentReportsToShow);
+    }
```

Comment:
@trjExpensify  Should we set a limitation to show for the workspace list similar to recent reports, or show all available workspace options like in this demo?

https://github.com/user-attachments/assets/d54da503-1898-44b7-ad3e-41da31eaecba

Reply:
I would think we'd always show all workspaces there... for most people it would be uncommon to have a ton of workspaces like you are showing above.

Reply:
Same, agreed!

Reply:
Thanks for the confirmation. @JKobrynski we can remove this condition.

Reply:
Done!